### PR TITLE
JDK-8283087: Create a test or JDK-4715503

### PR DESCRIPTION
**Pull request by srmandal**
Create a test or [JDK-4715503](https://bugs.openjdk.java.net/browse/JDK-4715503)

The getAccessibleContext method for JTable column header cells returns an
 AccessibleContext which cannot get the Bounding Rectangle for table header cells. The Bug causes problems with screen reader review mode.

The test added verifies that the are not exceptions and Bounding rectangles are returned for the Accessible Table Header Cells.

This review is for migrating tests from a closed test suite to open.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8283087](https://bugs.openjdk.java.net/browse/JDK-8283087): Create a test or JDK-4715503

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7807/head:pull/7807` \
`$ git checkout pull/7807`

Update a local copy of the PR: \
`$ git checkout pull/7807` \
`$ git pull https://git.openjdk.java.net/jdk pull/7807/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7807`

View PR using the GUI difftool: \
`$ git pr show -t 7807`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7807.diff">https://git.openjdk.java.net/jdk/pull/7807.diff</a>

</details>

### CODE REVIEW

File: test/jdk/javax/accessibility/4715503/AccessibleJTableCellBoundingRectangleTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            SwingUtilities.invokeAndWait(() -> createGUI());
+            Robot robot = new Robot();
+            robot.setAutoDelay(500);
+            robot.waitForIdle();
+
+            for (int i = 0; i <= colNames.length - 1; i++) {
+                try {
+                    System.out.println("Column " + i + " Bounds: "
+                        + jTable.getTableHeader().getAccessibleContext().getAccessibleChild(i)
+                        .getAccessibleContext().getAccessibleComponent().getBounds());
```
Comment (mrserb) [inline test/jdk/javax/accessibility/4715503/AccessibleJTableCellBoundingRectangleTest.java:59]: Can we add quick validation of the returned value? I assume it at least should not be null.

Comment (srmandal) [reply test/jdk/javax/accessibility/4715503/AccessibleJTableCellBoundingRectangleTest.java]: Fixed.

**Review by mrserb — APPROVED** [review]

**Comment by srmandal** [comment]
@mrserb Can you please sponsor my changes?

**Comment by mrserb** [comment]
@srmandal please confirm that the test works fine in the mach5.

**Comment by srmandal** [comment]
@mrserb Yes The test ran successfully on Mach5 with multiple runs (30) on windows-x64, linux-x64 and macos-x64.

**Pull request closed without merging**